app/controllers/product_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ProductController < ApplicationController
```

Comment:
This needs to be renamed to `ProductsController`

Reply:
fixed